doc/releases/changelog-dev.md

```diff
@@ -17,6 +17,35 @@
 
 <h3>Breaking changes</h3>
 
+* The `decomposition` method of operations has changed from a static
+  class method to an operation-dependent method.
+  [(#1873)](https://github.com/PennyLaneAI/pennylane/pull/1873)
+
+  Instead of the original syntax
+
+  ```python
+  >>> qml.CRX.decomposition(0.3, wires=[0, 1])
+  ```
+
+  the decomposition must be called on an instantiated version of the operation:
+
+  ```python
+  >>> qml.CRX(0.3, wires=[0, 1]).decomposition()
+  ```
```

Comment:
Just thinking that Josh is right, at some stage it would be nice to also make `_decomposition` outside-facing by giving it another name, but I don't think this is urgent for now

Reply:
While this is not urgent, I would say now _is_ the time to give it a better name, before we start building on it!

- The name doesn't exist yet, we should name it from the outset
- Attempting to change it later will be a bigger headache
- private attributes won't show up in the documentation, and developers will need to be able to see `_decomposition` in the docs